packages/rum/src/userActionCollection.ts

```diff
@@ -74,11 +74,16 @@ let currentUserAction: { id: string; startTime: number } | undefined
 export interface UserActionReference {
   id: string
 }
-export function getUserActionReference(time: number): UserActionReference | undefined {
-  if (currentUserAction && time >= currentUserAction.startTime) {
-    return { id: currentUserAction.id }
+export function getUserActionReference(time?: number): UserActionReference | undefined {
+  if (!currentUserAction) {
+    return undefined
   }
-  return undefined
+
+  if (time !== undefined && time < currentUserAction.startTime) {
+    return undefined
+  }
```

Comment:
what about grouping these checks?

Reply:
I did at first, but I think this makes the function more readable. WDYT?

Reply:
IMO:
```ts
if (!currentUserAction || (time !== undefined && time < currentUserAction.startTime)) {
```
has the same effect, you need understand both parts separately.
But having two if statements following each other with the same body bugged me a little, it does not seem very natural.
if you think that we should make the condition clearer, maybe extracting a function for the time predicate could clarify the intent.

Reply:
All right :)